engine/src/main/java/org/hibernate/search/query/dsl/sort/SortNativeContext.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public interface SortNativeContext extends SortAdditionalSortFieldContext, SortTermination {
```

Comment:
I guess you could persuade me to introduce a `SortNativeContext`, but I wasn't expecting us to provide such a thing.

Shouldn't we encourage people needing "native access" to avoid the DSL?

Do you think we should actually deprecate the old "setSort" and have all users - including the ones with specific native needs - move to this DSL?